openapi.yaml

```diff
@@ -1,279 +1,795 @@
+          example: -123.278729
+        geometry:
+          description: >-
+            an array of arrays of (latitude, longitude) coordinates 
+            representing the shapes that make up the location
+          properties:
+            type:
+              type: string
+              enum:
+                - Polygon
+                - MultiPolygon
+            coordinates:
+              description: Array of ring arrays for each physical structure of a location
+              type: array
+              items:
+                description: Array of points that make up a ring.
+                type: array
+                items:
+                  type: array
+                  items:
+                    type: number
+                    format: float
+        zoneSummary:
+          type: string
+          description: the description of the parking zone of the group
+          example: 'Zone: Austin Hall'
+        description:
+          type: object
+          description: miscellaneous information about the location in HTML and text format
+          properties:
+            html:
+              type: string
+              description: the description for the location, formatted using HTML tags
+              example: >-
+                <h4>\n\tAccessibility:</h4>
+                \n<p>\n\tENTRIES: North entry: ramp up to main floor.<br />
+                \n\tFLOORS: All floors; elevator (no Braille).</p>\n
+            text:
+              type: string
+              description: the desciption for the location, formatted without HTML tags
+              example: >-
+                \r\n\tAccessibility:\r\n\r\n\r\n\t
+                ENTRIES: North entry: ramp up to main floor.\r\n\t
+                FLOORS: All floors; elevator (no Braille).\r\n
+        address:
+          type: string
+          example: 12345 NE STREET RD
+          description: the street address of the location
+        city:
+          type: string
+          example: Corvallis
+          description: the city where the location is located
+        state:
+          type: string
+          example: OR
+          description: the state where the location is located
+        zip:
+          type: string
+          example: '97330'
+          description: the zip code of the location
+        telephone:
+          type: string
+          example: '541-111-1111'
+          description: a phone number that can be used to reach the location
+        fax:
+          type: string
+          example: '541-111-1111'
+          description: a fax number that can be used to reach the location
+        thumbnailUris:
+          type: array
+          description: an array of URLs for thumbnail images of the location
+          items:
             type: string
-            description: Name of pet
-            example: Hedwig
-          owner:
+            format: uri
+            example: https://example.oregonstate.edu/image.jpg
+        imageUris:
```

Comment:
Not sure can we merge these uris. Thoughts?

Reply:
Went for this:
```yaml
images:
          type: object
          description: full size and thumbnail image URLs for the location
          properties:
            fullSize:
              description: an array of URLs for images of the location
              type: array
              items:
                type: string
                format: uri
                example: https://example.oregonstate.edu/image.jpg
            thumbnails:
              type: array
              description: an array of URLs for thumbnail images of the location
              items:
                type: string
                format: uri
                example: https://example.oregonstate.edu/image.jpg
```

Let me know what you think.